frontend/web/components/modals/InvalidGithubIntegration.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { FC, useState } from 'react'
+import Utils from 'common/utils/utils'
+import InputGroup from 'components/base/forms/InputGroup'
+import Button from 'components/base/forms/Button'
+import { useUpdateGithubIntegrationMutation } from 'common/services/useGithubIntegration'
+import ErrorMessage from 'components/ErrorMessage'
+
+type InvalidGithubIntegrationType = {
+  onComplete?: () => void
+  organisationId: string
+  githubIntegrationId: string
+  errorMessage: string
+}
+
+const InvalidGithubIntegration: FC<InvalidGithubIntegrationType> = ({
+  errorMessage,
+  githubIntegrationId,
+  onComplete,
+  organisationId,
+}) => {
+  const [installationId, setInstallationId] = useState<string>('')
+
+  const [updateGithubIntegration, { isLoading: updating }] =
+    useUpdateGithubIntegrationMutation()
+
+  return (
+    <form
+      onSubmit={(e) => {
+        Utils.preventDefault(e)
+        updateGithubIntegration({
+          github_integration_id: githubIntegrationId,
+          installation_id: installationId,
+          organisation_id: organisationId,
+        }).then(() => {
+          onComplete?.()
+        })
+      }}
+    >
+      <div className='modal-body'>
+        <ErrorMessage error={errorMessage} />
+        <InputGroup
+          title='New Installation ID'
+          inputProps={{
+            className: 'full-width',
+            name: 'installationId',
+          }}
+          value={installationId}
+          onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
I really don't understand what this is doing, why would we be typing an installation id ?

Reply:
That's for the case they want to keep the existing links between PR/Issue and feature flags, while they updated for some reason the GH App Installation ID, i.e. they removed the integration from GH accidentally. Maybe too much for an edge case?